src/h5web/visualizations/line/ErrorBars.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { ReactElement } from 'react';
+import { useUpdate } from 'react-three-fiber';
+import { LineSegments } from 'react-three-fiber/components';
+import { BufferGeometry, Vector2, Vector3 } from 'three';
+import GlyphMaterial from './GlyphMaterial';
+import { GLYPH_URLS } from './models';
+
+interface Props {
+  capsPoints: Vector3[] | Vector2[];
+  barsSegments: Vector3[] | Vector2[];
+  color: string;
+  visible?: boolean;
+}
+
+function ErrorBars(props: Props): ReactElement {
+  const { barsSegments, capsPoints, color, visible } = props;
+
+  const barsGeometry = useUpdate<BufferGeometry>(
+    (geometry: BufferGeometry) => geometry.setFromPoints(barsSegments),
+    [barsSegments]
+  );
+  const capsGeometry = useUpdate<BufferGeometry>(
+    (geometry) => geometry.setFromPoints(capsPoints),
+    [capsPoints]
+  );
+
+  return (
+    <>
+      <LineSegments visible={visible}>
+        <lineBasicMaterial attach="material" color={color} linewidth={2} />
+        <bufferGeometry attach="geometry" ref={barsGeometry} />
+      </LineSegments>
+      <points visible={visible}>
+        <GlyphMaterial glyphURL={GLYPH_URLS.Cap} color={color} size={8} />
+        <bufferGeometry attach="geometry" ref={capsGeometry} />
+      </points>
+    </>
+  );
+}
+
+export default ErrorBars;
```

Comment:
Renamed `ErrorBarCurve` in `ErrorBars` so that we don't think that `DataCurve` and `ErrorBarCurve` are at the same level of hierarchy.

I only added the `useUpdate` (so that it takes points in props rather than the geometry) and removed the `Suspense` as it is already in the HOC